doc/src/manual/arrays.md

```diff
@@ -49,26 +49,26 @@ or a series of dimension sizes passed as a variable number of arguments.
-| [`trues(dims...)`](@ref)                    | a `Bool` array with all values `true`                                                                                                                                                                                                                    |
-| [`trues(A)`](@ref)                          | a `Bool` array with all values `true` and the shape of `A`                                                                                                                                                                                               |
-| [`falses(dims...)`](@ref)                   | a `Bool` array with all values `false`                                                                                                                                                                                                                   |
-| [`falses(A)`](@ref)                         | a `Bool` array with all values `false` and the shape of `A`                                                                                                                                                                                              |
-| [`reshape(A, dims...)`](@ref)               | an array with the same data as the given array, but with different dimensions.                                                                                                                                                                           |
+| [`zeros(T, dims...)`](@ref)                 | an array of all zeros of type `T`, the type defaults to `Float64` if not specified                                                                                                                                                                       |
+| [`zeros(A)`](@ref)                          | an array of all zeros with the same element type and shape as `A`                                                                                                                                                                                        |
+| [`ones(T, dims...)`](@ref)                  | an array of all ones of type `T`, the type defaults to `Float64` if not specified                                                                                                                                                                        |
+| [`ones(A)`](@ref)                           | an array of all ones with the same element type and shape as `A`                                                                                                                                                                                         |
+| [`trues(dims...)`](@ref)                    | a [`BitArray`](@ref) with all values `true`                                                                                                                                                                                                              |
```

Comment:
Hm, please ignore this suggestion :). (After reading the sentence you mentioned and scanning the table's unmodified parts, I agree that explicitly mentioning `dims` seems neither necessary nor consistent. Thanks!)